distributed/distributed.yaml

```diff
@@ -55,6 +55,10 @@ distributed:
         - distributed.http.proxy
         - distributed.http.statics
 
+    allowed-imports:
+      - dask
+      - distributed
```

Comment:
This list is somewhat restrictive right now. As mentioned in https://github.com/dask/distributed/pull/4140#discussion_r498847639 we could consider adding other libraries like xarray, cudf, etc. as needed. I'm okay leaving that for a follow up PR as necessary, but just wanted to re-iterate this point

Reply:
Agree